src/views/Maintenance.vue

```diff
@@ -10,7 +10,6 @@
 import { EventBus } from '../utils/eventBus';
 
 export default {
-  name: 'Maintenance',
```

Comment:
Is there any specific reason to remove implicit names for components and views?

Reply:
I suggest deleting them since we never refer to them by name. Plus, there is another opportunity to get a typo.